apps/storybook/stories/documentation/introduksjon.mdx

```diff
@@ -16,9 +16,14 @@ Designsystemet tilbyr komponenter til våre designere og utviklere, slik at de k
 
 ## Ta en titt 👩‍💻
 
-Du kan følge teamet på [confluence](https://kartverket.atlassian.net/wiki/spaces/TB/overview?homepageId=325713953) eller du kan benytte seg av [Håndboken i interaksjonsdesign](https://kartverket.atlassian.net/wiki/spaces/UX/overview) for nyttige tips.
+Du kan følge teamet på [confluence](https://kartverket.atlassian.net/wiki/spaces/TB/overview?homepageId=325713953) eller du kan benytte deg av [Håndboken i interaksjonsdesign](https://kartverket.atlassian.net/wiki/spaces/UX/overview) for nyttige tips.
 
-I [Figma](https://www.figma.com/file/j0OBUEuQZlrbQOV6ariHTo/Designsystem?type=design&node-id=0-1&t=Er7EiYPjkcDK55oR-0) finner du komponenter og tokens som er klar til bruk, med beskrivelser. Vi anbefaler at du er i kanalene #laug-design og i #gen-kvib på salck for å hold deg oppdatert🧠
+I [Figma](https://www.figma.com/file/j0OBUEuQZlrbQOV6ariHTo/Designsystem?type=design&node-id=0-1&t=Er7EiYPjkcDK55oR-0) finner du komponenter og tokens som er klare til bruk, med beskrivelser. Vi anbefaler at du er i kanalene #laug-design og i #gen-kvib på slack for å hold deg oppdatert🧠
```

Comment:
fjern "klare til bruk" for å ikke være misvisende; alle komponenter er nå klare til bruk uavhengig av om de er i figma